pkg/plugin/util/util.go

```diff
@@ -242,3 +242,29 @@ func ReplaceRegexInFile(path, match, replace string) error {
 	}
 	return nil
 }
+
+// OverwriteFile overwrites an existing file with `content`.
+func OverwriteFile(path, content string) error {
+	info, err := os.Stat(path)
+	if err != nil {
+		return err
+	}
+
+	err = ioutil.WriteFile(path, []byte(content), info.Mode())
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
```

Comment:
ihmo we do not need both methods, 
See that we can use instead: 
https://github.com/kubernetes-sigs/kubebuilder/blob/3ac65ec5a1f429f40f0fa81acc8a7d69a9a06bbc/pkg/plugin/util/util.go#L195-L216

Reply:
c/c @varshaprasad96